1.download-data/downloadData.py

```diff
@@ -0,0 +1,23 @@
+from scripts import downloadData
```

Comment:
```suggestion
import os
import argparse

from scripts import downloadData
```

Check out what [PEP8 says about imports](https://www.python.org/dev/peps/pep-0008/#imports):

> Imports should be grouped in the following order:
Standard library imports.
Related third party imports.
Local application/library specific imports.
You should put a blank line between each group of imports.